spec/testData/bulk_q.json

```diff
@@ -40,9 +40,9 @@
         }
       },
       "http://file.storage.server": {
-        "/file": {
+        "/": {
           "method": "PUT",
-          "response": "OKs"
```

Comment:
Looks like processing logic change

Reply:
accidentally was changed in the previous branch. Now reverted to the proper value - see master branch https://github.com/elasticio/salesforce-component/blob/master/spec/actions/bulk_q.json#L45